.env.test.example

```
@@ -0,0 +1,33 @@
+# 联调测试环境 env。复制为 `.env.test` 填真实值(不进版本库),compose 命令
+# 一律带 `--env-file .env.test`。与 prod 的 `.env` 完全独立:凭证、secret、
+# token 一律另起一套,绝不复用。
+
+# --- Postgres 容器(docker-compose.test.yml `insuredesk-db-test`) ----------
+TEST_POSTGRES_USER="insuredesk_test"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)